commons/commons-util/src/main/kotlin/ebi/ac/uk/io/Streams.kt

```diff
@@ -0,0 +1,5 @@
+package ebi.ac.uk.io
+
+inline fun <A : AutoCloseable, B : AutoCloseable> use(a: A, b: B, func: (A, B) -> Unit) {
```

Comment:
I think this method is not very clear at a single glane. Could we add some documentation?